test/src/module/command/restoreTest.c

```diff
@@ -1576,6 +1576,21 @@ testRun(void)
             "recovery_target_name = 'NAME'\n",
             "check recovery options");
 
+        // -------------------------------------------------------------------------------------------------------------------------
+        TEST_TITLE("recovery target lsn");
+
+        argList = strLstDup(argBaseList);
+        strLstAddZ(argList, "--type=lsn");
+        strLstAddZ(argList, "--target=LSN");
```

Comment:
For any new code we'd like to start using the new functions. You do not have to change it if you don't want to, it would just save me a step when I get to this file for updating all the tests to use the functions. 
```
hrnCfgArgRawZ(argList, cfgOptType, "lsn");
hrnCfgArgRawZ(argList, cfgOptTarget, "LSN");
```

Reply:
I would also prefer that the LSN look like a real LSN, e.g. `5218/5E35BBA8`. Just to make sure there are no issues passing `/` in an option value.

Reply:
Oops you're right, sorry, I should have thought about using the new tests functions.
Fixed both comments in 59dd292.